views/update/update-beneficial-owner-type.html

```diff
@@ -93,11 +93,14 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
   <div class="govuk-grid-column-two-thirds">
     <form method="post">
       {% include "includes/csrf_token.html" %}
-      {% include "includes/beneficial-owner-type-add-radios.html" %}
```

Comment:
I think this has undone some changes from main, shouldn't need any changes to this file